### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by digitalcoyote**
Added Chocolatey Install Instructions for #956

### CODE REVIEW

**Comment by tao12345666333** [comment]
Thanks for your work.

**Comment by digitalcoyote** [comment]
Chocolatey approved the package.
https://chocolatey.org/packages/kind

File: README.md [hunk 1]
```diff
@@ -54,6 +54,9 @@ On Windows:
 ```powershell
 curl.exe -Lo kind-windows-amd64.exe https://github.com/kubernetes-sigs/kind/releases/download/v0.5.1/kind-windows-amd64
 Move-Item .\kind-windows-amd64.exe c:\some-dir-in-your-PATH\kind.exe
+
+# OR via Chocolatey
```
Comment (tao12345666333) [inline README.md]: maybe we could add a link to https://chocolatey.org/

**Comment by BenTheElder** [comment]
/ok-to-test
Thanks for working on this

File: README.md (hunk 1 above)
Comment (BenTheElder) [reply README.md]: not: remove the `#`, this should not be an h1
Should also add this to the user guide.

**Comment by digitalcoyote** [comment]
Might be better to link directly to the package on chocolatey ( I started doing that for other projects after I made this PR).

Preference?

**Comment by BenTheElder** [comment]
SGTM 🙂

File: README.md (hunk 1 above)
Comment (digitalcoyote) [reply README.md]: That should display as a powershell comment. Would you prefer it was separated from the code block for installing on Windows?

Comment (BenTheElder) [reply README.md]: 🤦‍♂ I missed the ``` above, thanks

**Comment by tao12345666333** [comment]
SGTM :+1:

**Comment by BenTheElder** [comment]
LGTM, thanks!
with v0.6.0 coming this week, what will we need to do? I'd like to list install methods that will point users to that version during kubecon :sweat_smile: 
xref https://github.com/kubernetes-sigs/kind/issues/956#issuecomment-552628536

**Comment by digitalcoyote** [comment]
Haven't had much time to look today. Where is the User Guide? I'll try to update it with the relevant instructions as well.

**Comment by digitalcoyote** [comment]
> LGTM, thanks!
> with v0.6.0 coming this week, what will we need to do? I'd like to list install methods that will point users to that version during kubecon 😅
> xref [#956 (comment)](https://github.com/kubernetes-sigs/kind/issues/956#issuecomment-552628536)

I also answered in the original issue, but it's automatic as long as it's using my CI scripts (scrapes the releases page for a version and download URL's and inserts them into the proper places for the package). I'd be happy to assist in integrating it into your release process if you prefer though.

**Comment by BenTheElder** [comment]
https://github.com/kubernetes-sigs/kind/blob/master/site/content/docs/user/quick-start.md

Not a big deal, we can sync in a follow up.
It has the same content for this part.

**Comment by BenTheElder** [comment]
> I also answered in the original issue, but it's automatic as long as it's using my CI scripts (scrapes the releases page for a version and download URL's and inserts them into the proper places for the package). I'd be happy to assist in integrating it into your release process if you prefer though.

that sounds great already, thanks! :-)

**Comment by tao12345666333** [comment]
> it's automatic as long as it's using my CI scripts (scrapes the releases page for a version and download URL's and inserts them into the proper places for the package)

Thanks for your work! kind v0.6.0 has been released https://github.com/kubernetes-sigs/kind/releases/tag/v0.6.0 
I want to know how often your script is executed？

**Comment by digitalcoyote** [comment]
> > it's automatic as long as it's using my CI scripts (scrapes the releases page for a version and download URL's and inserts them into the proper places for the package)
> 
> Thanks for your work! kind v0.6.0 has been released https://github.com/kubernetes-sigs/kind/releases/tag/v0.6.0
> I want to know how often your script is executed？

It runs twice daily. Once it is pushed it also has to go through moderation before Chocolatet will list the package. That moderation is done by volunteers, so depending on how many moderators are reviewing, and how many packages they have ahead of it, it can take a bit to be available (there are arguments for Chocolatey that will update to an unlisted version,but they are not recommended)

**Comment by BenTheElder** [comment]
thanks!

we might normalize the styling later with the rest of the content, but this is an awesome net contribution, thanks again!

skipping running tests again for a docs change...

**Pull request merged**